docs/DeveloperGuide.md

```diff
@@ -323,7 +323,7 @@ This feature was implemented so that it is easy for the user to change a guest's
 
 <span style="font-size: large; color: #e46c0a">Implementation:</span>
 * The `edit` command takes in an `INDEX` indicating the guest to edit in the current panel (starting from 1)
-  and 8 optional fields (`Name`, `Phone`, `Email`, `Room`, `Date Range`, `Number Of Guests`, `Is Room Clean` and `Request`)
+  and 8 optional fields (`name`, `phone`, `email`, `room`, `dateRange`, `numberOfGuests`, `isRoomClean` and `request`)
```

Comment:
Should you change this for the add as well?

Reply:
Yes changed!